Directory.Build.props

```diff
@@ -1,6 +1,8 @@
 <Project>
   <PropertyGroup>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
These are orthogonal changes to encore code style and ensure all future project include this and have nullable enabled.